### PR TITLE
Add Screenly OSE digital display support

### PR DESCRIPTION
**Pull request by k7hpn**

### CODE REVIEW

**Review by madison-boman — COMMENTED** [review]
I thought it looked good overall! Aside from the noted files, I have these comments:

- When you go to edit/add the objects in ContentManagement, the associated tab is not active marked as active.

- I think the "provision" button might need a little more context around it, unless theres going to be documented instruction. It gives off a little intimidation of clicking it, if you didn't know it's full context.

File: src/Ops.Controllers/Areas/ContentManagement/DigitalDisplaysController.cs [hunk 1]
```diff
@@ -0,0 +1,607 @@
...
+                    update.Success = false;
+                    update.Message = $"There was an error updating the asset set: {oex.Message}";
+                }
+            }
+            return Json(update);
+        }
+
+        [HttpGet]
+        [Route("[action]/{displayId}")]
+        [SaveModelState]
```
Comment (iafb) [inline src/Ops.Controllers/Areas/ContentManagement/DigitalDisplaysController.cs:384]: GET actions should use the RestoreModelState attribute

File: src/Ops.Controllers/Areas/ContentManagement/DigitalDisplaysController.cs [hunk 2]
```diff
@@ -0,0 +1,607 @@
...
+            {
+                DigitalDisplay = await _digitalDisplayService.GetDisplayAsync(displayId),
+                DigitalDisplaySets = SetDropDown(await _digitalDisplayService.GetSetsAsync()),
+                Locations = LocationDropDown(await _locationService.GetAllLocationsAsync()),
+            });
+        }
+
+        [HttpPost]
+        [Route("[action]/{displayId}")]
+        [RestoreModelState]
```
Comment (iafb) [inline src/Ops.Controllers/Areas/ContentManagement/DigitalDisplaysController.cs:398]: POST actions should use the SaveModelState attribute

File: src/Ops.Controllers/Areas/ContentManagement/DigitalDisplaysController.cs [hunk 3]
```diff
@@ -0,0 +1,607 @@
...
+                return RedirectToAction(nameof(Index));
+            }
+
+            return RedirectToAction(nameof(UpdateDisplay),
+                new { displayId = viewmodel.DigitalDisplay.Id });
+        }
+
+        [HttpGet]
+        [Route("[action]")]
+        [SaveModelState]
```
Comment (iafb) [inline src/Ops.Controllers/Areas/ContentManagement/DigitalDisplaysController.cs:419]: GET actions should use the RestoreModelState attribute

**Comment by madison-boman** [comment]
> I thought it looked good overall! Aside from the noted files, I have these comments:
> 
> * When you go to edit/add the objects in ContentManagement, the associated tab is not active marked as active.
> * I think the "provision" button might need a little more context around it, unless theres going to be documented instruction. It gives off a little intimidation of clicking it, if you didn't know it's full context.

Disregard the last bullet. It was fixed in PR Standardize Ops UI; add segment-only layout cloning #293

**Pull request merged**